src/pykiso/connector.py

```diff
@@ -163,6 +163,25 @@ def _cc_receive(self, timeout: float, raw: bool = False) -> MsgType:
         pass
 
 
+class ProxyCChannel(CChannel):
+    def __init__(self, **kwargs) -> None:
+        """constructor.
+
+        :param kwargs: named arguments
+        """
+        super().__init__(**kwargs)
+
+    @abc.abstractmethod
+    def cc_share(self, *args: tuple, **kwargs: dict) -> None:
+        """How to deal with requests parmeters from external sources
```

Comment:
```suggestion
        """How to deal with requests parameters from external sources
```

Reply:
This is only used for the proxy right? So I would raise an error here instead of a pass

Reply:
Its an abstract method it will raise an error if you don't define it.

Reply:
My bad ^^